### ISSUE TITLE
CocoaPods badges are shown as 'not found'

### ISSUE THREAD

**Issue by vhuseinova-msft** [badges/shields#8439]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

The shields.io badges for AzureCommunicationUICalling CocoaPods version, license and platform worked before as expected, but started to show "not found" recently.

<img width="365" alt="Screen Shot 2022-09-26 at 11 57 44 AM" src="https://user-images.githubusercontent.com/98852890/192357937-af15a598-2a9c-4d35-9aba-434e2106385b.png">

AzureCommunicationUICalling podspec is available [here](https://github.com/CocoaPods/Specs/blob/master/Specs/2/8/d/AzureCommunicationUICalling/1.0.0/AzureCommunicationUICalling.podspec.json) and it's parsable on CocoaPods website.

<img width="1196" alt="Screen Shot 2022-09-26 at 11 54 06 AM" src="https://user-images.githubusercontent.com/98852890/192357605-897551a2-1cbf-4258-904f-afd2ea869635.png">

### 🔗 Link to the badge

https://img.shields.io/cocoapods/l/AzureCommunicationUICalling
https://img.shields.io/cocoapods/v/AzureCommunicationUICalling
https://img.shields.io/cocoapods/p/AzureCommunicationUICalling

**Comment by chris48s** [comment]
Hi. I think this is an issue on the CocoaPods side rather than us.

We are calling https://trunk.cocoapods.org/api/v1/pods/AzureCommunicationUICalling/specs/latest which redirects us to a spec file that doesn't exist: https://raw.githubusercontent.com/CocoaPods/Specs/08beb7948001739710fcb35bd6ed797fba3ff0bd/Specs/2/8/d/AzureCommunicationUICalling/1.0.0/AzureCommunicationUICalling.podspec.json

Side note: if I call https://trunk.cocoapods.org/api/v1/pods/AzureCommunicationUICalling/specs/1.0.0 that redirects us to a spec file that does exist: https://raw.githubusercontent.com/CocoaPods/Specs/06244b445d06dd97ae4744db57c7901aadc2997f/Specs/2/8/d/AzureCommunicationUICalling/1.0.0/AzureCommunicationUICalling.podspec.json

Can you try raising an issue with them about it.

I have seen this issue happen once before: https://github.com/badges/shields/issues/6596 but I have no idea what the resolution was. If you get this sorted out upstream with CocoaPods, could you link us to the issue or let us know what happened in case it comes up again.

Cheers

**Comment by vhuseinova-msft** [comment]
Thank you @chris48s!
The issue is created for CocoaPods and can be found [here](https://github.com/CocoaPods/cocoapods-trunk/issues/176).

**Comment by vhuseinova-msft** [comment]
Hi @chris48s! A new version of the pod was released yesterday and the `latest` API returns the correct podspec because of this. I'll still keep the issue for CocoaPods team open to follow up on the incorrect latest response.

I noticed that version and platform were available yesterday after the release and only licence badge was available today in the morning but it looks like all used bages (version, licence and platform) aren't available now (even on [shield.io](https://shields.io)):

<img width="551" alt="Screen Shot 2022-10-04 at 9 07 37 AM" src="https://user-images.githubusercontent.com/98852890/193870905-03cd4c64-ad2e-4583-96ff-032bd6604d72.png">

<img width="468" alt="Screen Shot 2022-10-04 at 9 07 59 AM" src="https://user-images.githubusercontent.com/98852890/193870929-b615ba93-1b9f-4376-9c52-276602081089.png">
